.azure-pipelines/client.yml

```diff
@@ -128,7 +139,13 @@ jobs:
     displayName: 'Setup and Run Tests'
     inputs:
       scriptPath: 'scripts/devops_tasks/setup_execute_tests.py'
-      arguments: '"$(build_targeting_string)"'
+      arguments: '"$(build_targeting_string)" --junitxml="junit/test-results.xml"'
+    
+  - task: PublishTestResults@2
+    condition: succeededOrFailed()
+    inputs:
+      testResultsFiles: '**/test-*.xml'
```

Comment:
You set up a specific input above in the python script call, can you reference it specifically as well?